### PR TITLE
add source to widgets, move layerselector to cartoassets

### PR DESCRIPTION
**Pull request by matallo**
re: https://github.com/CartoDB/cartodb/issues/9956

this PR adds the source to the widgets in the Builders, the time-series widget has not been updated, but I'd tackle this as a separate issue, as part of the [Time-series milestone](https://github.com/CartoDB/cartodb/milestone/138)

pageshot of untitled map - title map carto 2017-06-03-1629 32

work done in several repos:

- [x] cartoassets: https://github.com/CartoDB/CartoAssets/pull/166
- [x] cartodb.js: https://github.com/CartoDB/cartodb.js/pull/1680
- [x] deep-insights.js: https://github.com/CartoDB/deep-insights.js/pull/552

and the work in this PR itself

this can be tested in ded03

**acceptance**

- add several sources and analyses and check the letter and color in the source match, as well as the source type

- the widgets source shouldn't appear in the embeds

### CODE REVIEW

**Comment by saleiva** [comment]
I think there is a typo on the acceptance description. The widgets source should be only visible within buiilder, so we shouldn't show them in any public view of the map.

**Comment by matallo** [comment]
indeed there is @saleiva :'D

File: app/assets/stylesheets/editor-3/entry.scss [hunk 1]
```diff
@@ -34,7 +34,6 @@
 @import 'infobox';
 @import 'infowindow';
 @import 'inline-editor';
-@import 'layer-selector';
```
Comment (nobuti) [inline app/assets/stylesheets/editor-3/entry.scss]: we should delete this file too.

**Comment by nobuti** [comment]
It works wonderfully. Let's wait until testing issues are fixed to deploy them.

**Comment by xavijam** [comment]
Tests are not passing guyis :S.

**Comment by matallo** [comment]
working on it, another problem appeared
```
>> Error: Cannot find module "." at
```

this has only to do with this branch, master branch tests are passing https://github.com/CartoDB/cartodb/pull/12434

**Comment by arianaescobar** [comment]
Acceptance from Design:

- [ ] The source type is not showing up under the widget title on the widget options

screen shot 2017-07-06 at 15 00 55

- [ ] There's a 5px difference of vertical spacing between those two items. It should be 5px below.

source

- [ ] Source type should also be visible in the widget list

screen shot 2017-07-06 at 14 54 53

- [ ] Could the source type be a bit darker? Right now is not as easy to distinguish from the source name than on the mockup.

artboard

Beyond that, everything looks 👌🏼🌈

**Comment by matallo** [comment]
Styles in widget options are out of the scope of this issue, this is for the widgets 😛 (but feel free to report them in a separate issue)

Shall we really darken the color? it is the same than the analysis typography in the layers cards, in that case maybe we should darken them both

screen shot 2017-07-06 at 15 10 06

**Comment by arianaescobar** [comment]
Ok, opening up a new ticket for those other changes. 

Regarding the color, yeah, let's darken it. Since we show it in regular on the widgets, and not in semibold like on the cards, we need it to have more contrast.

**Comment by arianaescobar** [comment]
Perfect @matallo, approved by design ™️

**Pull request merged**